docs/ref/applications.txt

```diff
@@ -56,25 +56,29 @@ application and have models, etc. (which would require adding it to
 Configuring applications
 ========================
 
-To configure an application, subclass :class:`~django.apps.AppConfig` and put
-the dotted path to that subclass in :setting:`INSTALLED_APPS`.
+To configure an application, create an ``apps.py`` module inside the
+application, then define a subclass of :class:`AppConfig` there.
 
 When :setting:`INSTALLED_APPS` contains the dotted path to an application
-module, Django checks for a ``default_app_config`` variable in that module.
+module, by default, if Django finds exactly one :class:`AppConfig`
+subclass in the ``apps.py`` submodule, it uses that configuration for the
+application. This behavior may be controlled by setting
+:attr:`AppConfig.default` to ``False``.
```

Comment:
This behavior may be **disabled**?

I find "controlled" insufficiently specific there.